### PR TITLE
Fixes #9240 - Cell separators visible/not full width on initial load

### LINKED ISSUE #9240: Password and last modified rows separators are shown and delete item row separator is not full width on initial saved login detail load
## Steps to reproduce
1. Tap the menu button
2. Tap Passwords
3. Tap a saved login
4. Swipe down to view control center or background Firefox
5. Dismiss control center or foreground Firefox

### Expected behavior
- Password and last modified row separators should be hidden on initial load (as is when returning from background)
- Delete item row separator should be full width on initial load (as is when returning from background)

### Actual behavior
- Password and last modified row separators are visible on initial load, but hidden when Firefox returns to foreground
- Delete item row separator is not full width on initial load, but is full width when Firefox returns to foreground

### Device & build information
* Device: iPhone 12 Pro Max
* Simulators: iPhone 13 Pro, iPhone 13 Pro Max
* Build version: [85d8ceb](https://github.com/mozilla-mobile/firefox-ios/commit/85d8ceb33321b0697157357706eecbd4f66812fa)

### Notes
Attachments: 
Steps to reproduce:
steps

Initial load:
initial-load

Restore from background:
restored-from-background

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3244)

### PR DESCRIPTION
**Pull request by michaelbiv**
Fixes: #9240 

Moved the cell separator adjustments out of viewDidLayoutSubviews as these adjustments were being overridden by subsequent calls to tableView:cellForRowAt.  I can update the function declaration syntax I used if it doesn't match desired patterns or move to an extension if desired as well.

fix

Note:  The last modified string cutoff was resolved in #9239

### CODE REVIEW

File: Client/Frontend/Settings/LoginDetailViewController.swift [hunk 1]
```diff
@@ -224,6 +203,17 @@ extension LoginDetailViewController: UITableViewDataSource {
         loginCell.delegate = self
         return loginCell
     }
+    
+    fileprivate func setCellSeparatorHidden(_ cell: UITableViewCell) {
+        // Prevent seperator from showing by pushing it off screen by the width of the cell
+        cell.separatorInset = UIEdgeInsets(top: 0, left: cell.bounds.width, bottom: 0, right: 0)
```
Comment (michaelbiv) [inline Client/Frontend/Settings/LoginDetailViewController.swift:209]: See #9239 : This should be `cell?.separatorInset = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: cell?.bounds.width ?? 0)` if #9239 is accepted

Comment (lmarceau) [reply Client/Frontend/Settings/LoginDetailViewController.swift:209]: @michaelbiv This needs to be updated then before we can merge this? #9239 was merged.

Comment (michaelbiv) [reply Client/Frontend/Settings/LoginDetailViewController.swift:209]: Ok, I'll get it updated

Comment (lmarceau) [reply Client/Frontend/Settings/LoginDetailViewController.swift:209]: @michaelbiv any update on this?

**Comment by electricRGB** [comment]
Rebased and reopened in #11274

**Pull request closed without merging**